code/game/machinery/hologram.dm

```diff
@@ -704,3 +704,80 @@ For the other part of the code, check silicon say.dm. Particularly robot talk.*/
 
 #undef HOLOPAD_PASSIVE_POWER_USAGE
 #undef HOLOGRAM_POWER_USAGE
+
+/obj/machinery/holopad/proc/create_admin_hologram(client/C)
```

Comment:
This entire proc is very expensive.

Reply:
in fairness it can only be triggered by admins